.github/workflows/build.yml

```diff
@@ -567,7 +507,7 @@ jobs:
         key: ${{ matrix.os }}-multissl-openssl-${{ env.OPENSSL_VER }}
     - name: Install OpenSSL
       if: steps.cache-openssl.outputs.cache-hit != 'true'
```

Comment:
With `steps.cache-openssl` here, we won't be hitting the `cache-ssl` entry right? maybe it's better to store a cache hit with `steps.cache-${{ matrix.ssl }}` if it's possible?